hazelcast/src/main/java/com/hazelcast/internal/management/events/ReloadFinishedEvent.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.management.events;
+
+import com.hazelcast.internal.json.JsonObject;
+
+public class ReloadFinishedEvent extends AbstractEventBase implements ReloadEvent {
+    private final String reloadResponseMessage;
+
+    public ReloadFinishedEvent(String reloadResponseMessage) {
```

Comment:
can we have a proper JSON as `reloadReponseMessage`?

Reply:
https://github.com/hazelcast/hazelcast/pull/20260/commits/3d693d1919f8b882dd89ab1681feb4d85fa58f53